content/en/docs/instrumentation/python/exporters.md

```diff
@@ -0,0 +1,197 @@
+---
+title: "Exporters"
+weight: 4
+---
+
+In order to visualize and analyze your traces and metrics, you will need to
+export them to a backend.
+
+## Console exporter
+
+The console exporter is useful for development and debugging tasks, and is the
+simplest to set up.
+
+```python
+from opentelemetry import trace
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor, ConsoleSpanExporter
+from opentelemetry.sdk.resources import SERVICE_NAME, Resource
+
+# Service name is required for most backends
+resource = Resource(attributes={
+    "service.name": "your-service-name"
+})
+
+provider = TracerProvider(resource=resource)
+processor = BatchSpanProcessor(ConsoleSpanExporter())
+provider.add_span_processor(processor)
+trace.set_tracer_provider(provider)
+
+# Merrily go about tracing!
+```
+
+## OTLP endpoint or Collector
+
+To send data to an OTLP endpoint or the OpenTelemetry Collector, you'll want to
+configure an OTLP exporter that sends to your endpoint.
+
+First, install an OTLP exporter:
+
+```
+pip install opentelemetry-exporter-otlp-proto-http
+```
+
+Then you can use it when you initialize tracing:
+
+```python
+from opentelemetry import trace
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+from opentelemetry.sdk.resources import SERVICE_NAME, Resource
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+
+# Service name is required for most backends
+resource = Resource(attributes={
+    SERVICE_NAME: "your-service-name"
+})
+
+provider = TracerProvider(resource=resource)
+processor = BatchSpanProcessor(OTLPSpanExporter(endpoint="your-endpoint-here"))
+provider.add_span_processor(processor)
+trace.set_tracer_provider(provider)
+
+
+# Merrily go about tracing!
+```
+
+### Using gRPC
+
+If you'd prefer to use gRPC, you can install the package:
+
+```
+pip install opentelemetry-exporter-otlp-proto-grpc
+```
+
+And replace the `OTLPSpanExporter` import declaration with the following:
+
+```python
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+```
+
+## Jaeger
+
+If you are using [Jaeger](https://www.jaegertracing.io/) to visualize trace
+data, you'll need to set it up first. This is how to run it in a docker
+container:
+
+```shell
```

Comment:
Just as a note for a follow up issue: this same code block is used at multiple places, would be great to externalise it with something like the `code_block_from` macro